packages/angular_devkit/build_webpack/src/browser/schema.json

```diff
@@ -29,7 +29,6 @@
     },
     "tsConfig": {
       "type": "string",
-      "default": "tsconfig.app.json",
```

Comment:
Considering this is required, shouldn't that still be there?

Reply:
They should not have defaults precisely because they are required. Their absence breaks the build.

If they had a default, they should not be required. The build would also fail while running because a file does not exist, instead of failing validation.

I think failing validation is better than failing at runtime with a message about a file that is not mentioned anywhere in the project proper.

Reply:
I see. The default is wrong. Got it.